CARTOGRAPHY.md

```diff
@@ -1,39 +1,49 @@
-# Cartography
 
-This is a style that serves multiple purposes, so here are some guidelines when considering cartographic changes.
+# Design goals and guidelines for this style
 
-## Purposes
+This is an attempt to outline the goals of this style and the principles under 
+which the maintainers make decisions.  These rules are not set in stone, they 
+can change and they may not be followed in all cases but contributors should 
+be able to expect they are generally the guiding principles design wise.
 
-There are multiple primary purposes of the map style, which pull in different directions
+It does not make much sense to try following these principles blindly as a 
+contributor without understanding them, they are meant to guide you to develop 
+an intuition and understanding how to make design decisions to fit into the 
+overall concept of this style.
 
-* It's the primary feedback mechanism for mappers to validate their edits - so detail is useful
-* It's a major part of the impression visitors to osm.org receive - so clear design is useful
-* It's an examplar stylesheet for rendering OSM data - so easy customisation is useful
+## General purpose
 
-It must always be borne in mind that a map style cannot show every detail of the OSM data, and in many cases it is more appropriate to show the detail in other, more specialist styles.
+This style has multiple purposes:
 
-## Colours
+* It's an important feedback mechanism for mappers to validate their edits.
+* It's a major part of the public face of OpenStreetMap, for many people the map on osm.org rendered with this style _is_ OpenStreetMap.
+* It's used in many map applications as a general purpose map.
+* It's an examplar stylesheet for rendering OSM data.
```

Comment:
I would add here that openstreet-map is plays an important part in tag usage behaviour of mappers and in tag hygiene.